plugins/flatpak/gs-flatpak.c

```diff
@@ -3935,11 +4145,19 @@ gs_flatpak_finalize (GObject *object)
 		self->changed_id = 0;
 	}
 
+	if (self->mount_removed_handler_id > 0) {
```

Comment:
`mount_removed_handler_id` never seems to be set. Was this supposed to be for the `mount-removed` signal on `self->volume_monitor` (not `self->monitor`)?

Reply:
Yeah. I was late for picking up my daguther :/

Reply:
I don’t mind if a PR is delayed by a few hours so you can pick your daughter up and then test the PR confidently before submitting it!